build/lib/builtInExtensions.js

```diff
@@ -16,10 +16,11 @@ const vfs = require('vinyl-fs');
 const ext = require('./extensions');
 const fancyLog = require('fancy-log');
 const ansiColors = require('ansi-colors');
+const product =  require('vs/platform/product/common/product');
```

Comment:
Did you test this? I'm not sure it's going to work - this is ran separately as a build-time task. Not within the ADS process.

https://github.com/microsoft/azuredatastudio/blob/7ebd1eb053c479a35a4b046fa00bbf5c6526aa99/scripts/sql.bat#L21

Reply:
So reading the process.env as discussed offline now. Thanks!